File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV2003U00GrdINV2004Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV2003U00GrdINV2004Info
    {
        private String _pkinv2004;
        private String _fkinv2003;
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _chulgoQty;
        private String _chulgoDanuiName;
        private String _chulgoDanga;
        private String _remark;
        private String _rowState;

        public String Pkinv2004
        {
            get { return this._pkinv2004; }
            set { this._pkinv2004 = value; }
        }

        public String Fkinv2003
        {
            get { return this._fkinv2003; }
            set { this._fkinv2003 = value; }
        }

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String ChulgoQty
        {
            get { return this._chulgoQty; }
            set { this._chulgoQty = value; }
        }

        public String ChulgoDanuiName
        {
            get { return this._chulgoDanuiName; }
            set { this._chulgoDanuiName = value; }
        }

        public String ChulgoDanga
        {
            get { return this._chulgoDanga; }
            set { this._chulgoDanga = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public INV2003U00GrdINV2004Info() { }

        public INV2003U00GrdINV2004Info(String pkinv2004, String fkinv2003, String jaeryoCode, String jaeryoName, String chulgoQty, String chulgoDanuiName, String chulgoDanga, String remark, String rowState)
        {
            this._pkinv2004 = pkinv2004;
            this._fkinv2003 = fkinv2003;
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._chulgoQty = chulgoQty;
            this._chulgoDanuiName = chulgoDanuiName;
            this._chulgoDanga = chulgoDanga;
            this._remark = remark;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV2003U00GrdINV2004Info.java
package nta.med.data.model.ihis.invs;

public class INV2003U00GrdINV2004Info {
	private Double pkinv2004;
	private Double fkinv2003;
	private String jaeryoCode;
	private String jaeryoName;
	private Double chulgoQty;
	private String chulgoDanuiName;
	private Double chulgoDanga;
	private String remark;


	public INV2003U00GrdINV2004Info(Double pkinv2004, Double fkinv2003, String jaeryoCode, String jaeryoName, Double chulgoQty, String chulgoDanuiName, Double chulgoDanga, String remark) {
		super();
		this.pkinv2004 = pkinv2004;
		this.fkinv2003 = fkinv2003;
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.chulgoQty = chulgoQty;
		this.chulgoDanuiName = chulgoDanuiName;
		this.chulgoDanga = chulgoDanga;
		this.remark = remark;
				
	}


	public Double getPkinv2004() {
		return pkinv2004;
	}


	public void setPkinv2004(Double pkinv2004) {
		this.pkinv2004 = pkinv2004;
	}


	public Double getFkinv2003() {
		return fkinv2003;
	}


	public void setFkinv2003(Double fkinv2003) {
		this.fkinv2003 = fkinv2003;
	}


	public String getJaeryoCode() {
		return jaeryoCode;
	}


	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}


	public String getJaeryoName() {
		return jaeryoName;
	}


	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}


	public Double getChulgoQty() {
		return chulgoQty;
	}


	public void setChulgoQty(Double chulgoQty) {
		this.chulgoQty = chulgoQty;
	}


	public String getChulgoDanuiName() {
		return chulgoDanuiName;
	}


	public void setChulgoDanuiName(String chulgoDanuiName) {
		this.chulgoDanuiName = chulgoDanuiName;
	}


	public Double getChulgoDanga() {
		return chulgoDanga;
	}


	public void setChulgoDanga(Double chulgoDanga) {
		this.chulgoDanga = chulgoDanga;
	}


	public String getRemark() {
		return remark;
	}


	public void setRemark(String remark) {
		this.remark = remark;
	}

	
}
